chat-plugins/help-ai.js

```diff
@@ -0,0 +1,100 @@
+		} else if (this.includes('lfg') || this.includes('battle me') || this.includes('challenge me')) {
+			return `If you're looking for somebody to play a battle with, select the tier you wish to play on the dropdown to the left (on mobile, go to "Home") and select your team (if not a randomly generated tier). This will match you with a random opponent whose also playing on the ladder`;
+		} else if (this.includes('rate my team') || (this.includes('rmt') && !roomLink)) {
+			return `If you'd like for someone to rate your team, we have a room dedicated for this named <a href="/rmt">Rate My Team</a>. Please be sure to read their roomintro for a guide on how to use the room and its' rules before chatting there.`;
+		} else if ((this.includes('get') || this.includes('become')) && (this.includes('rank') || this.includes('staff') || this.includes('voice') || this.includes('driver') || this.includes('mod'))) {
+			return `You can read the <a href="${STAFF_FAQ}">Staff FAQ</a> for more information about how to become a staff member.`;
+		} else if ((this.includes('inappropriate') || this.includes('bad') || this.includes('offensive') || this.includes('racist')) && (this.includes('username') || this.includes('nick') || this.includes('name'))) {
+			return `To report someone using an inappropriate username, you can PM a Moderator (@) in ${tarRoom}.`;
+		} else if ((this.includes('appeal') || this.includes('for no reason')) && !roomLink) {
+			return `To appeal a room punishment, PM an authority currently online in that room (use <code>/roomauth [roomname]</code>; bold names are people currently online.)<br />` +
+				`To appeal a lock or global ban, you can either PM the authority that locked you, and/or post an appeal in the <a href="${APPEALS}">Appeals Forum</a>. To do so, you will have to have (or create) a Smogon Forums account.`;
+		} else if (this.includes('change') && (this.includes('pass') || this.includes('pw'))) {
+			return `To change your password, use the <button name="openOptions" class="button"><i style="font-size: 16px ; vertical-align: -1px" class="fa fa-cog"></i> Options</button> button and click "Password Change".`;
+		} else if (command) {
+			return `Hmm, I'm not sure about that one. Try asking in the Help room.`;
+		} else {
+			return false;
+		}
+	}
+	getResponseHTML(question, command, room) {
+		const autoReply = this.ask(question, command, room);
+		if (autoReply) {
+			let buff = '';
+			buff += Chat.html`<strong>Question:</strong> ${question}<br />`;
+			buff += `<strong>Answer:</strong> ${autoReply}`;
+			if (!command) buff += `<br /><br /><button class="button" name="send" value="HELP: ${question}">This did not answer my question; ask the Help room</button>`;
+			return buff;
+		} else {
+			return false;
+		}
+	}
+	includes(text) {
```

Comment:
``this.includes`` is shorter than ``question.includes``, and I'm thinking of making this function smarter later